app/web/features/search/SearchResultsList.tsx

```diff
@@ -48,6 +48,8 @@ const useStyles = makeStyles((theme) => ({
     backgroundColor: theme.palette.background.paper,
     boxShadow: "0 0 4px rgba(0,0,0,0.25)",
     marginBottom: theme.spacing(3),
+    scrollMarginTop: "200px",
```

Comment:
We have a variable for the navbar height you can use actually rather than hardcoding it to `200px` which might be too far down on mobile, for example.

You can use `theme.shape.navPaddingXs` for smalls screens and `theme.shape.navPaddingSmUp` for larger.

Try searching those variables in the codebase in your IDE and you'll see some examples of how to handle the different screen sizes.